setup.py

```diff
@@ -64,6 +64,9 @@
       ],
       test_suite='test',
       extras_require={
+          'brotli': [
+              'brotlipy',
```

Comment:
Can we at least set a minimum bound here?

Reply:
I recommend >=0.5.1 so we can use `brotli.Error`. See: https://github.com/python-hyper/brotlipy/blob/master/HISTORY.rst#051-2016-08-17

Reply:
Done

Reply:
Sorry @immerrr I edited my comment as you pushed. Would you mind increasing to 0.6.0?

Reply:
Sure